### DISCUSSION TITLE
Time of events is wrong by 2 hours off

### DISCUSSION THREAD

**Discussion by mrHJS** [bitfireAT/icsx5#167] [General]
Hi there,

I try to add subscribe an ics file, which works, but the time of the events is always 2 hours of, so instead of 9am, ICSX5 show 11am.
The timezone in the ics is set to Europe/Berlin and the times in the ics file are correct. MacOS and iPhone do show the correct times when I subscribe to the calendar. Also Google Calendar online show the correct times.
Only events synced via ICSX5 show wrong times.

Any ideas what I can do?

Thank you!

**Comment by mrHJS** [comment]
When I remove the "Z" from the ics file's dates and times, I get the correct time which then is local time.

**Comment by ArnyminerZ** [comment]
Z stands for UTC in Java's Datetime. This means that even though your ICS file might declare the timezone to be Berlin, dates are specified in UTC. So even though you think the dates are correct in the ICS file, they are converted to UTC when synchronization runs.

It's 2 hours off since Berlin's timezone is GMT+2, which means that the time specified is 2 hours less than Berlin's one.

This is correct and intended behavior, I don't know why it's working fine on Apple's products.

If you have any other issue, or don't understand something, ask and I'll try to explain it better 😉
